### PR TITLE
New buttons

### PR DESCRIPTION
**Pull request by Mafatik**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        super(Cell, self).__init__(**kwargs)
+        self.on_press = self.x_add
+    
+    def x_add(self):
+        self.text = "X"
+
+class Field(GridLayout):
+
+    def __init__(self, **kwargs):
+        super(Field, self).__init__(**kwargs)
```
[the remark this answers is not in the document]
Comment (Mafatik) [reply main.py]: super uses to implement the functionality of the original overloaded class. kwargs** uses to transfer undefined number of positional or named arguments (Kivy libraries requires)

[the remark this answers is not in the document]
Comment (TanyaAkulich) [reply main.py]: **kwargs not kivy only. But why do you need this string in the method

[the remark this answers is not in the document]
Comment (Mafatik) [reply main.py]: Know that. I need it cause kivy shoud be able to transfer all arguments it needs

[the remark this answers is not in the document]
Comment (GitH3ll) [reply main.py]: Ebat vy chely anglichane

**Pull request closed without merging**